apps/internal/base/base.go

```diff
@@ -219,6 +221,7 @@ func New(clientID string, authorityURI string, token *oauth.Client, options ...O
 		cacheAccessor: noopCacheAccessor{},
 		manager:       storage.New(token),
 		pmanager:      storage.NewPartitionedManager(token),
+		mu:            &sync.RWMutex{},
```

Comment:
Is the generic name `mu` to imply that this must be held when reading/writing any field?

Reply:
No, this is only for locking around `cacheAccessor`. Would something like `cacheMu` be better?

Reply:
In that case, the pattern is to name them `<var that's locked>Mu` so `cacheAccessorMu`